HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -39,6 +39,119 @@ def customiseForOffline(process):
 
     return process
 
+def customizeHLTfor47611(process):
+    """ Add CAGeometry ESProducer"""
+    
+    if not hasattr(process, 'HLTRecoPixelTracksSequence'):
+        return process
+        
+    ca_producers = ['CAHitNtupletAlpakaPhase1@alpaka','alpaka_serial_sync::CAHitNtupletAlpakaPhase1']
+
+    ca_parameters = [ 'CAThetaCutBarrel', 'CAThetaCutForward', 
+        'dcaCutInnerTriplet', 'dcaCutOuterTriplet', 
+        'doPtCut', 'doZ0Cut', 'idealConditions', 
+        'includeJumpingForwardDoublets', 'phiCuts','doClusterCut','CPE'] 
+
+    process.hltCAGeometry = cms.ESProducer('CAGeometryESProducer@alpaka',
+        caDCACuts = cms.vdouble(
+            0.15, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0.25
+        ),
```

Comment:
Why are these (collections of) parameters explicitly listed here in the HLT customisation function, instead of using fillDescriptions for this?

Reply:
Since CAGeometryESProducer is a new plugin, fillDescriptions can take care of any/all parameters!